src/components/ha-alert.ts

```diff
@@ -115,6 +115,9 @@ class HaAlert extends LitElement {
       margin-right: 8px;
       width: 24px;
     }
+    slot[name="icon"]::slotted(*) {
```

Comment:
Wouldn't it be better to change the `issue-type::before` to be `issue-type::after` ?

Reply:
`::after` makes it worse, then it also skews svg icons that are slotted

Reply:
Ok. This still seems a bit hacky because slotted content can override their own CSS. Is it possible to either try `z-index` to put it lower or to have the slotted content fit inside a `<div>` that has position relative ?